<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="ErrorEventArgs.xml" source-language="en-US" target-language="zh-CN">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac58f2921fa15912959397b39597816d57dfaa68afe.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">8f2921fa15912959397b39597816d57dfaa68afe</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.IO.ErrorEventArgs">
          <source>Provides data for the <ph id="ph1">&lt;see cref="E:System.IO.FileSystemWatcher.Error" /&gt;</ph> event.</source>
          <target state="translated">为 <ph id="ph1">&lt;see cref="E:System.IO.FileSystemWatcher.Error" /&gt;</ph> 事件提供数据。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.ErrorEventArgs">
          <source><ph id="ph1">&lt;xref:System.IO.ErrorEventArgs&gt;</ph> contains the <ph id="ph2">&lt;xref:System.Exception&gt;</ph> that caused the <ph id="ph3">&lt;xref:System.IO.FileSystemWatcher.Error?displayProperty=nameWithType&gt;</ph> event.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.IO.ErrorEventArgs&gt;</ph> 包含<ph id="ph2">&lt;xref:System.Exception&gt;</ph>导致<ph id="ph3">&lt;xref:System.IO.FileSystemWatcher.Error?displayProperty=nameWithType&gt;</ph>事件。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.ErrorEventArgs">
          <source>This class provides the <ph id="ph1">&lt;xref:System.IO.ErrorEventArgs.GetException%2A&gt;</ph> method to retrieve the exception.</source>
          <target state="translated">此类提供<ph id="ph1">&lt;xref:System.IO.ErrorEventArgs.GetException%2A&gt;</ph>方法来检索该异常。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.ErrorEventArgs">
          <source>The following example creates a new instance of <ph id="ph1">&lt;xref:System.IO.ErrorEventArgs&gt;</ph> and initializes it with an <ph id="ph2">&lt;xref:System.Exception&gt;</ph>.</source>
          <target state="translated">下面的示例创建的新实例<ph id="ph1">&lt;xref:System.IO.ErrorEventArgs&gt;</ph>并初始化它与<ph id="ph2">&lt;xref:System.Exception&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.ErrorEventArgs">
          <source>Then the example calls <ph id="ph1">&lt;xref:System.IO.ErrorEventArgs.GetException%2A&gt;</ph> to retrieve the <ph id="ph2">&lt;xref:System.Exception&gt;</ph> and display the error message.</source>
          <target state="translated">然后，该示例调用<ph id="ph1">&lt;xref:System.IO.ErrorEventArgs.GetException%2A&gt;</ph>检索<ph id="ph2">&lt;xref:System.Exception&gt;</ph>并显示错误消息。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.ErrorEventArgs">
          <source>There is no form associated with this code.</source>
          <target state="translated">没有与此代码没有任何窗体。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.ErrorEventArgs">
          <source>The following example shows how to create a FileSystemWatcher to monitor file changes (creates, deletes, renames, changes) occurring on a disk drive.</source>
          <target state="translated">下面的示例演示如何创建 FileSystemWatcher 来监视文件的更改 （创建、 删除、 重命名、 更改） 磁盘驱动器上发生。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.ErrorEventArgs">
          <source>The example also shows how to properly receive error notifications.</source>
          <target state="translated">该示例还演示如何正确接收错误通知。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.IO.ErrorEventArgs.#ctor(System.Exception)">
          <source>An <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that represents the error that occurred.</source>
          <target state="translated">表示已发生的错误的 <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.IO.ErrorEventArgs.#ctor(System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.ErrorEventArgs" /&gt;</ph> class.</source>
          <target state="translated">初始化 <ph id="ph1">&lt;see cref="T:System.IO.ErrorEventArgs" /&gt;</ph> 类的新实例。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.ErrorEventArgs.#ctor(System.Exception)">
          <source>The following example creates a new instance of <ph id="ph1">&lt;xref:System.IO.ErrorEventArgs&gt;</ph> and initializes it with an <ph id="ph2">&lt;xref:System.Exception&gt;</ph>.</source>
          <target state="translated">下面的示例创建的新实例<ph id="ph1">&lt;xref:System.IO.ErrorEventArgs&gt;</ph>并初始化它与<ph id="ph2">&lt;xref:System.Exception&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.ErrorEventArgs.#ctor(System.Exception)">
          <source>Then the example calls <ph id="ph1">&lt;xref:System.IO.ErrorEventArgs.GetException%2A&gt;</ph> to retrieve the <ph id="ph2">&lt;xref:System.Exception&gt;</ph> and display the error message.</source>
          <target state="translated">然后，该示例调用<ph id="ph1">&lt;xref:System.IO.ErrorEventArgs.GetException%2A&gt;</ph>检索<ph id="ph2">&lt;xref:System.Exception&gt;</ph>并显示错误消息。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.ErrorEventArgs.#ctor(System.Exception)">
          <source>There is no form associated with this code.</source>
          <target state="translated">没有与此代码没有任何窗体。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.IO.ErrorEventArgs.GetException">
          <source>Gets the <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that represents the error that occurred.</source>
          <target state="translated">获取表示已发生的错误的 <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.IO.ErrorEventArgs.GetException">
          <source>An <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that represents the error that occurred.</source>
          <target state="translated">表示已发生的错误的 <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.ErrorEventArgs.GetException">
          <source>The following example creates a new instance of <ph id="ph1">&lt;xref:System.IO.ErrorEventArgs&gt;</ph> and initializes it with an <ph id="ph2">&lt;xref:System.Exception&gt;</ph>.</source>
          <target state="translated">下面的示例创建的新实例<ph id="ph1">&lt;xref:System.IO.ErrorEventArgs&gt;</ph>并初始化它与<ph id="ph2">&lt;xref:System.Exception&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.ErrorEventArgs.GetException">
          <source>Then the example calls <ph id="ph1">&lt;xref:System.IO.ErrorEventArgs.GetException%2A&gt;</ph> to retrieve the <ph id="ph2">&lt;xref:System.Exception&gt;</ph> and display the error message.</source>
          <target state="translated">然后，该示例调用<ph id="ph1">&lt;xref:System.IO.ErrorEventArgs.GetException%2A&gt;</ph>检索<ph id="ph2">&lt;xref:System.Exception&gt;</ph>并显示错误消息。</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.ErrorEventArgs.GetException">
          <source>There is no form associated with this code.</source>
          <target state="translated">没有与此代码没有任何窗体。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>